pages/api/reset-password.js

```diff
@@ -0,0 +1,34 @@
+import withContainer from "../../src/middleware/withContainer";
```

Comment:
Could you add a unit test spec for this API please?

Reply:
have added this 🙂